src/coreclr/ilasm/GrammarExtractor/Program.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using static Patterns;
+
+if (args.Length != 1)
+{
+    Console.Error.WriteLine("Usage: <file>");
```

Comment:
Is this script supposed to be executed on src\coreclr\ilasm\asmparse.y file? It does not work for me - it fails with "Could not find %% markers" when I try to run it against that file.

Reply:
Now they produce identical results as perl. I refactored it using source generator and didn't re-tested it. 🤦‍♂️

Reply:
Thanks - it works for me know. The output of the tool is apparently checked in as `src\coreclr\ilasm\prebuilt\asmparse.grammar`. It would be nice to fix the tool and/or the checked in output to match.

Reply:
Updated. Perl script was missing the license header and asmparse.grammar was missing the latest update `_V` variant. (which suggests it was previously updated by hand)

Reply:
Updated. Perl script was missing the license header and asmparse.grammar was missing the  `_V` in int tokens. (which suggests it was previously updated by hand)